.github/workflows/test.yaml

```diff
@@ -676,12 +676,12 @@ jobs:
             rules: 'main'
           - group: ubuntu-no-lts
```

Comment:
@sergiocazzolato could this say "ubuntu-not-lts", "no" is grammatically wrong here (and elsewhere)

Reply:
I propose we use the Ubuntu terminology here "Development vs LTS". As per `do-release-upgrade -d` and https://canonical-ubuntu-packaging-guide.readthedocs-hosted.com/en/latest/explanation/releases/ etc...

Reply:
I think we should have:
ubuntu-devel: which contains the development version
ubuntu-no-lts: which contains the last no lts version which is still supported.

@bboozzoo do you want to update that? This will require a change in the required systems which will require everyone do a rebase. Otherwise we can make it in a following pr.